Jenkinsfile

```diff
@@ -130,11 +133,22 @@ podTemplate(cloud: 'openshift', label: 'coreos-assembler', yaml: pod, defaultCon
             prevBuildID = utils.shwrap_capture("readlink builds/latest")
         }
 
+        def parent_version = ""
+        def parent_commit = ""
         stage('Build') {
+            def parent_arg = ""
+            if (utils.path_exists("tmp/releases.json")) {
+                parent_commit = utils.shwrap_capture("""
+                    jq -r '.releases[-1].commits[] | select(.architecture == "${basearch}") | .checksum' tmp/releases.json
```

Comment:
This seems OK, but we could also parse the JSON directly in Groovy which seems less awkward.